bundles/org.openhab.binding.touchwand/src/main/java/org/openhab/binding/touchwand/internal/dto/Csc.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.touchwand.internal.dto;
+
+/**
+ * The {@link Csc} implements Csc data class.
+ *
+ * @author Roie Geron - Initial contribution
+ */
+public class Csc {
```

Comment:
Can you give this class a better name?

Reply:
I agree it is a bad name .
The class name is from the Json exposed to me from TouchWand controller and its name is to match the Json field .
This is out of my control , I prefer to keep it this way.